defaults:
  - mppi: dingo_push
  - isaacgym: push
  - _self_

goal: [2.0, 2.0]
render: true
n_steps: 5
nx: 6

actors: ['dingo']
initial_actor_positions: [[0.0, 0.0, 0.05]]
